packages/js-sdk/test/getConfig.test.js

```diff
@@ -10,10 +10,5 @@ contract("SuperfluidSDK JS SDK", () => {
         // defaultConfig
         const defaultConfig = SuperfluidSDK.getConfig(8888);
         assert.isUndefined(defaultConfig.resolverAddress);
-        // test environment
```

Comment:
please revert this one and see if it works

Reply:
changes reverted!